### ISSUE TITLE
Flatpak version crashes on launch

### ISSUE THREAD

**Issue by pizzadude** [JMS55/sandbox#1]
Hi, the flatpak version crashes on launch. Here's the log from the terminal:

```
flatpak run com.github.jms55.Sandbox
INTEL-MESA: warning: Haswell Vulkan support is incomplete
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/video_recorder.rs:47:39
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

**Comment by JMS55** [comment]
Hi, Sandbox uses the wgpu-rs library for rendering, which itself uses Vulkan/Metal/Directx12, but notably NOT OpenGL yet. On linux, Vulkan is the only option for now. Haswell cpu's do not entirely support Vulkan, and so probably can't run Sandbox. You can watch the tracking issue for OpenGl support here https://github.com/gfx-rs/wgpu/issues/450.

That said, it appears it's crashing unrelated to rendering. For some reason, it can't seem to find XDG_VIDEOS_DIR, which is where Sandbox stores video recordings. That shouldn't happen in a flatpak I don't think, but I can add a fallback directory sometime today.

**Comment by pizzadude** [comment]
I fixed it by editing ~/.config/user-dirs.dirs and changing XDG_VIDEOS_DIR="$HOME/" to XDG_VIDEOS_DIR="$HOME/Videos". It seems to be running now.

**Comment by pizzadude** [comment]
![image](https://user-images.githubusercontent.com/1454420/87460559-0cb63980-c5db-11ea-804c-a236f15fd9e7.png)
Seems to work fine even without the full Vulkan support.

**Comment by JMS55** [comment]
Glad it works even on Haswell cpu's. 
Fixed by 7962931ee6982f5c598b9ded0bcfcb8d6b921fd8
